interlok-core/src/main/java/com/adaptris/core/CloneMessageServiceList.java

```diff
@@ -61,6 +66,10 @@ public class CloneMessageServiceList extends ServiceListBase {
   @Valid
   private MetadataFilter overrideMetadataFilter;
 
+  @Getter
```

Comment:
A small javadoc explanation on what it is would be nice here.
Also you could use @InputFieldDefault(value = "false") so the UI is aware of the default value and can display that to users.